public/js/notes.js

```diff
@@ -1,7 +1,7 @@
 let tableBody = document.getElementById("tableBody");
 
 // push the data in localstorage
-function updateNote() {
+function fetchNote() {
```

Comment:
Are you fetching only 1 note from local storage?

Reply:
No